docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -133,12 +133,12 @@ To query audio device information:
             }
         }
 
-        AudioManager.DeviceStateChanged += OnDeviceStateChanged;
+        AudioManager.DeviceRunningChanged += OnDeviceRunningChanged;
         ```
 
 
         > **Note**   
-		> The initial state of the connected device is `Deactivated`.
+		> The initial running state of the connected device is `false', that is 'not running'.
```

Comment:
There is an tag error.  The code tag is \`  \`.

How about the following sentence?
```
> The initial running state of the connected device is `false`, which means the connected device is not running.
```

Reply:
Fixed